pkg/localcontroller/managers/incrementallearning/incrementallearningjob.go

```diff
@@ -256,6 +259,70 @@ func (im *Manager) evalTask(job *Job) error {
 	return nil
 }
 
+// deployTaskII starts deploy task when job supports hot model update
+func (im *Manager) deployTaskII(job *Job) error {
+	var localModelConfigFile string
+	if v, ok := job.ObjectMeta.Annotations[runtime.ModelHotUpdateAnnotationsKey]; ok {
```

Comment:
change value of `ModelHotUpdateAnnotationsKey` from "sedna.io/model-hot-update" => "sedna.io/model-hot-update-config"

Reply:
ok